tests/Unit/NumericalAlgorithms/Spectral/Test_Filtering.cpp

```diff
@@ -80,4 +80,55 @@ SPECTRE_TEST_CASE("Unit.Numerical.Spectral.ExponentialFilter",
     }
   }
 }
+
+template <Spectral::Basis BasisType, Spectral::Quadrature QuadratureType>
+void test_zero_lowest_modes() noexcept {
+  CAPTURE(BasisType);
+  CAPTURE(QuadratureType);
+  for (size_t num_pts =
+           Spectral::minimum_number_of_points<BasisType, QuadratureType>;
+       num_pts < Spectral::maximum_number_of_points<BasisType>; ++num_pts) {
```

Comment:
The documentation of `maximum_number_of_points` says it's the max allowed, so shouldn't this be an `<=`? This comment applies in previously-existing code in this test, too, so maybe left for a different cleanup.